### DISCUSSION TITLE
Change HomeKit Secure Video Local Copy location?

### DISCUSSION THREAD

**Discussion by podupuis** [koush/scrypted#322] [General]
Any way to change the location where local HomeKit secure video is saved? I would love to change it to an external drive. Thanks!

**Comment by Luk141888** [comment]
I'd like that, too. Or turn it off completely.

**Comment by tellok** [comment]
If you are using docker scrypted, you can bind  container's `/server/volume/plugins/@scrypted/homekit/files/hksv` to wherever you want, this is directory containing the recordings.
If you are not using docker, maybe you can try the symlinks.
